tests/test_skyWcs.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import absolute_import, division, print_function
+import unittest
+
+import lsst.utils.tests
+from lsst.afw.geom import SkyWcs, SpherePoint, Extent2D, Point2D, degrees, arcseconds, makeCdMatrix
+from lsst.afw.geom.testUtils import TransformTestBaseClass
+
+
+class TanSkyWcsTestCase(TransformTestBaseClass):
+    """Test pure TAN constructor of SkyWcs
+    """
+
+    def setUp(self):
+        TransformTestBaseClass.setUp(self)
+        self.crpix = Point2D(100, 100)
+        self.crval = SpherePoint(30 * degrees, 45 * degrees)
```

Comment:
Good point. Testing at the poles will require some extra work, but testing near the poles and at or near RA=0 can use the existing code.